watsong/spotify.py

```diff
@@ -243,7 +242,7 @@ def create_playlist(songs: List[Song]) -> str:
             collaborative=True,
             description="A playlist created by watsong just for you",
         )
-    sp.playlist_replace_items(playlist["id"], [song["uri"] for song in songs])
+    sp.playlist_replace_items(playlist["id"], [song["uri"] for song in songs][:100])
```

Comment:
I think this has issues with not removing songs if there are fewer than 100 songs, but I haven't been able to reproduce. Talk on Tuesday?

Reply:
I think it should work, we can talk